packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -2,17 +2,44 @@
  * CSS Custom Properties
  *
  * These properties can be overridden using the component's tag as selector.
- *
- * @prop --calcite-panel-content-space: Specifies the padding of the component's content.
- * @prop --calcite-panel-footer-padding: Specifies the padding of the component's footer.
- * @prop --calcite-panel-header-border-block-end: Specifies the component header's block end border.
+
+ * @prop --calcite-panel-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-panel-shadow: Specifies the shadow of the component.
+ * @prop --calcite-panel-heading-text-color: Specifies the text color of the component's `heading`.
+ * @prop --calcite-panel-description-text-color: Specifies the text color of the component's `description`.
+
  * @prop --calcite-panel-background-color: Specifies the background color of the component.
+ * @prop --calcite-panel-header-background-color: Specifies the background color of the component's footer.
+ * @prop --calcite-panel-footer-background-color: Specifies the background color of the component's footer.
+ * @prop --calcite-panel-content-top-background-color: Specifies the border color of the `"content-top"` slot.
+ * @prop --calcite-panel-content-bottom-background-color: Specifies the border color of the `"content-bottom"` slot.
```

Comment:
Just to clarify - there is padding rendered around the slotted content. This means that a user cannot actually control the background, padding, or border color of these areas (and why Action Bar has no corresponding background color or padding props, because there is no UI surface a user can't control).

I don't understand why we'd add these for some of the content areas and not others - if a user wants to theme the header and footer it is a logical next request to theme these values.

Reply:
tokens are meant to describe our design patterns. This does not represent our design patterns. If a user needs this and can provide a reasonable use-case for it then we can consider it but until then it's just adding extra bulk